CMakeLists.txt

```diff
@@ -57,14 +57,14 @@ source_group ("Sources" FILES ${AssimpJSSourceFiles})
 
 if (${EMSCRIPTEN})
 	add_executable (AssimpJS ${AssimpJSSourceFiles})
-	target_compile_options (AssimpJS PUBLIC "$<$<CONFIG:Debug>:-gsource-map>")
-	target_compile_options (AssimpJS PUBLIC -fexceptions)
-
-	target_link_options (AssimpJS PUBLIC -sMODULARIZE=1)
-	target_link_options (AssimpJS PUBLIC -sEXPORT_NAME='assimpjs')
-	target_link_options (AssimpJS PUBLIC -sSTACK_SIZE=10MB)
-	target_link_options (AssimpJS PUBLIC -sALLOW_MEMORY_GROWTH=1 --no-heap-copy)
-	target_link_options (AssimpJS PUBLIC -sNO_DISABLE_EXCEPTION_CATCHING)
+	# target_compile_options (AssimpJS PUBLIC "$<$<CONFIG:Debug>:-gsource-map>")
```

Comment:
Why do you think these are not needed?

Reply:
I deactivated the link_options because it is not building.
I'm getting this error for example:
em++: error: Attempt to set a non-existent setting: 'STACK_SIZE'
 - did you mean one of STACK_BASE, ASYNCIFY_STACK_SIZE, STACK_MAX?
 - perhaps a typo in emcc's  -s X=Y  notation?
 Seems like this option is invalid. Please check this out.